Ottimizzazione traduzione Impostazioni, Musica, Sicurezza e Fix vari

Caledor opened the pull request:
Ottimizzata e fixata la traduzione delle Impostazioni (inteso come tutto ciò che era raggiungibile dalle impostazioni), a meno di Schermata di Blocco, Notifiche, Spazio secondario e modalità semplificata e cambiamenti propagati ai singoli device (compresi quelli mancanti dalla pr precedente).
Ottimizzate Musica e Sicurezza.
Propagato Cit.apk a virgo leo e cancro, che erano ancora alla release iniziale

Fix minori vari, come:
Fixati i due punti cinesi in occidentali (： -> :)
Fixati tutti i typo di maiuscole (es: MOdalità)

@ale8530 
Alcune parti sono state saltate (schermata di blocco etc) perché lavorandoci mi sono accorto che alcuni cambiamenti erano puramente stilistici e quindi discutibili. Non volevo che le eventuali discussioni che ne uscivano complicassero una pr più grande come questa e quindi ho deciso che te li sottoporrò a parte in seguito, una pr alla volta, in modo che siano molto più facili da revisionare e discutere.

Anche stavolta il consiglio è di partire dalla fine per la revisione, visto che tutti i cambiamenti fatti sui singoli device sono copia carbone di quelli fatti nel main.

Vista la mole di roba da controllare, prenditi tutto il tempo che ti serve, mi preoccuperò io di integrare i cambiamenti che verranno fatti nel frattempo al repository principale.

In caso avessi una perplessità su un qualsiasi cambiamento, commentalo anche solo con una parola e provvederò a motivartelo.

Extra: ho notato un caso particolare di incoerenza. XiaomiSimActivateService.apk e XiaomiServiceFramework.apk hanno moltissime stringhe in comune ma localizzate in modo diverso. Ne ho fixate forse 1 o 2 ovvie con questa pr, ma per moltissime altre non avrei saputo quale era la migliore non avendole mai incontrate.

Caledor committed:
Hotfix propagazione

propaga alcuni cambiamenti già presenti nella PR ai singoli dispositivi

Caledor committed:
Ottimizzazione impostazioni 3 (Personali)

Caledor committed:
Ottimizzazione Impostazioni 4 (Non disturbare, batteria, memoria)

Caledor committed:
Ottimizzazione impostazioni 5

Tutto il resto tranne: Schermata di blocco, Notifiche, Spazio secondario
e la sezione Dispositivo

Caledor committed:
Fine ottimizazione impostazioni, Musica e Sicurezza

Rimane l'ottimizzazione di Schermata di blocco, notifiche, spazio
secondario, permessi e installazione pacchetti

Caledor committed:
Fix minori e Musica

Caledor committed:
Fix

La stringa modificata a riga 633 andrebbe in plurals

Caledor committed:
Merge remote-tracking branch 'refs/remotes/MishFdM/master'

Caledor committed:
Fix

Caledor committed:
Propagazione parte 1

Caledor committed:
Propagazione Settings finale

Caledor committed:
Aggiornamento Cit.apk device

I Cit.apk di leo virgo e cancro erano ancora alla prima release

ale8530 commented:
@Mishfdm deve effettuare la revisione per le troppe stringhe coinvolte

ale8530 commented on Italian/main/SecurityCenter.apk/res/values-it/strings.xml:
```diff
@@ -660,7 +660,7 @@ I dati rimanenti includono dati che sono stati erogati il mese precedente."</str
     <string name="quick_optimize_no_issue_subtitle">Tutto sembra in buono stato</string>
     <string name="quick_optimize_no_issue">Nessun problema batteria rilevato</string>
-    <string name="quick_optimize_sub_summary_text">Aggiunto %s di vita alla batteria</string>
+    <string name="quick_optimize_sub_summary_text">Aggiunti %sdi vita alla batteria</string>
```
Qui c'è un errore correggere

"Aggiunti %s di vita alla batteria" va bene come impostato da noi, ripristinare stringa precedente

Caledor replied:
se non mi sfugge qualcosa è meglio così perché lo spazio è già presente nella stringa che viene inserita al posto di %s. la versione originale causava un doppio spazio. La stringa inglese inoltre ha lo stesso formato della mia (senza spazio) immagino per lo stesso motivo

ale8530 replied:
Allora ok

ale8530 replied:
:+1:

ale8530 commented on Italian/main/Calendar.apk/res/values-it/strings.xml:
```diff
@@ -316,10 +316,10 @@
-    <string name="setting_about_miui_version">Versione MIUI:</string>
-    <string name="setting_about_calendar_version">Versione calendario:</string>
-    <string name="setting_about_imei">IMEI:</string>
+    <string name="setting_about_summary">Versione: </string>
```
C'è un problema di spazio?

utilizzare il codice 

"Versione MIUI:\u0020"
Su tutte e quattro le stringhe

Caledor replied:
844c13851bd280aa137767843cbc5caca45f7d4d

ale8530 commented on Italian/main/CleanMaster.apk/res/values-it/strings.xml:
```diff
@@ -101,7 +101,7 @@
     <string name="hints_large_file_header_left">File grossi</string>
     <string name="hints_installed_apps_header_left">App installate</string>
-    <string name="toast_resolve_apk_error">Risolvi errori APK</string>
+    <string name="toast_resolve_apk_error">Risolvi errori pacchetti</string>
```
Lasciare APK

Caledor replied:
cambiati per coerenza, visto che da altre parti veniva usato pacchetti. Dovrei portare tutto ad APK invece?

ale8530 replied:
Per chi utilizza android "APK" è più giusto, propongo alla prossima PR di cambiare questa cosa

ale8530 commented on Italian/main/CleanMaster.apk/res/values-it/strings.xml:
```diff
@@ -150,7 +150,7 @@
     <string name="hints_quick_cleanning">Pulizia…</string>
     <string name="hints_quick_prepare">Preparazione…</string>
-    <string name="start_downloading_app">Avvia download：%1$s</string>
+    <string name="start_downloading_app">Avvia download： %1$s</string>
```
Togliere lo spazio va bene stringa originale

Caledor replied:
844c13851bd280aa137767843cbc5caca45f7d4d

ale8530 commented on Italian/main/CleanMaster.apk/res/values-it/strings.xml:
```diff
@@ -389,12 +389,12 @@
-    <string name="activity_title_deep_apk">Elimina APK</string>
-    <string name="deepclean_apk_content">Elimina APK o App non necessarie</string>
-    <string name="deepclean_apk_summary_empty">Tutto OK, nessun APK da eliminare</string>
+    <string name="activity_title_deep_apk">Elimina pacchetti</string>
```
Reimpostare a tutti APK

ale8530 commented on Italian/main/MiuiVideo.apk/res/values-it/strings.xml:
```diff
@@ -440,8 +440,8 @@
     <string name="lastrefresh">Ultimo aggiornamento:</string>
-    <string name="version_select">Selezione versione</string>
-    <string name="version_select_summary">Seleziona versione Alpha, Developer o Stabile</string>
+    <string name="version_select">Selezione versione </string>
```
Sempre codice \u0020 per dare lo spazio a fine o inizio stringa

Caledor replied:
844c13851bd280aa137767843cbc5caca45f7d4d

ale8530 commented on Italian/main/TeleService.apk/res/values-it/strings.xml:
```diff
@@ -1006,8 +1006,8 @@ Sarà usata al suo posto la rete preferita.</string>
-    <string name="auto_answer_scenario_enteries_bluetooth_only">Solo bluetooth</string>
-    <string name="auto_answer_scenario_enteries_headset_bluetooth">Solo auricolari o bluetooth</string>
+    <string name="auto_answer_scenario_enteries_bluetooth_only">Solo Bluetooth</string>
+    <string name="auto_answer_scenario_enteries_headset_bluetooth">Solo auricolari o Bluetooth</string>
```
Bluetooth  in maiuscolo non ha senso, dovresti mettere anche "Auricolari" , quindi ripristinare minuscolo a Bluetooth grazie

Caledor replied:
Bluetooth credo vada sempre maiuscolo in quanto nome proprio, inoltre mi pare che queste fossero le uniche 2 stringhe in cui era minuscolo

ale8530 replied:
Fai un controllo con "Bluetooth" su tutta la repo e vedi se effettivamente erano minuscole le altre

Caledor replied:
Fatto. Non sono gli unici ma c'è una prevalenza schiacciante di Bluetooth maiuscoli: 139 a 21 contando solo la cartella main e solo quelli all'interno della frase (insomma nessuna maiuscola delle 139 è giustificata dal fatto di essere a inizio frase). E controllando i sorgenti di android sul git di google ho notato che in settings.apk per esempio non esistono minuscoli. Approfitto e passo tutti in maiuscola?

ale8530 replied:
allora mettiamo "Solo auricolari" in "Solo Auricolari" almeno è coerente la stringa se no è brutto vedere auricolari e Bluetooth

Caledor replied:
Oddio a me sembra un po' azzardata come mossa, e ci vedo più contro che pro.
Potrei capire il cambiamento dettato da esigenze estetiche, ma a parte il fatto che è abbastanza soggettivo, è contrario alle regole dell'italiano e crea un pericoloso precedente, che potrebbe motivare altri innumerevoli e minuscoli cambiamenti simili.
Per questi motivi ti direi che è molto molto meglio evitare, ma se proprio non ti va giù un compromesso meno vistoso potrebbe essere invertirli in "Solo Bluetooth o auricolari"

ale8530 replied:
Lasciamo la tua stringa allora Dante Alighieri 👍 

Scherzo eh ;-)

ale8530 replied:
👍

Caledor replied:
ahahah figurati. è solo che essendo praticamente un hobby per me questo della traduzione non è la prima volta che mi capita di sentire questo tipo di richieste e avendo già sperimentato le conseguenze che possono avere ho preferito metterti in guardia subito ;)

ale8530 commented on Italian/main/Settings.apk/res/values-it/strings.xml:
```diff
@@ -3955,22 +3955,22 @@ ed esegui il software."</string>
     <string name="screen">Schermo</string>
     <string name="system">Sistema</string>
-    <string name="screen_color_temperature">Temperatura colori</string>
+    <string name="screen_color_temperature">Colori</string>
```
Temperatura colori è più adeguato ripristinare precedente traduzione

Caledor replied:
"Temperatura di colori" è praticamente una sinestesia sulla sinestesia, tant'è che si dice "il colore è caldo" e mai "la temperatura del colore è X", per questo ho preferito cambiare.
Una versione alternativa (e probabilmente migliore ora che ho approfondito) può essere "Temperatura di colore" o "Temperatura colore", che è una cosa ben definita e già usata come stringa per esempio in ubuntu nelle impostazioni dello schermo

ale8530 replied:
OK va bene per "Temperatura colore"

ale8530 commented on Italian/main/CloudService.apk/res/values-it/strings.xml:
```diff
@@ -478,8 +478,8 @@ Ultimo backup: %2$s"</string>
     <string name="micloud_request_permission_content">Mi Cloud ha necessità dei permessi per la sincronizzazione dei dati. Consenti a Mi Cloud di accedere a Calendario, Contatti, Mi Account, stato SIM, registro chiamate e messaggi.</string>
     <string name="micloud_request_permission_confirm">Ho capito</string>
     <string name="short_cut_desc">"Vuoi aggiungere l'icona Mi Cloud alla home?"</string>
-    <string name="add_short_cut">Aggiungi</string>
```
Ripristinare "Aggiungi"

Caledor replied:
il problema era che "aggiungi" andava a capo perché costretto a stare nello spazio in cui entrava "Add". Essendo il tasto in risposta alla domanda di sopra (480), ho adattato in "Sì"

ale8530 replied:
Ok qui se hai verificato sul dispositivo

ale8530 replied:
:+1:

Caledor committed:
Fix pr

ale8530 commented:
Quelle ok ti ho messo il simbolo ":+1: "

ale8530 commented:
Fare gli ultimi fix richiesti, dei commenti rimasti aperti, cosi possiamo fare il merge

Caledor commented:
avendo rimandato pacchetti/apk è rimasto solo temperatura colore o mi sono perso qualcosa?

ale8530 commented:
Mancano solo quelle due cose :)

Caledor commented:
Quindi il ritorno da pacchetti a apk non lo vuoi in una prossima PR ma adesso? Per conferma, non vorrei aver capito male

ale8530 commented:
Sistemare in questa

Caledor commented:
Ok arriva. Ah, riguardo invece i bluetooth minuscoli, approfitto già che ci sono e li metto tutti maiuscoli?

ale8530 commented:
Su quella ti ho messo 👍 quindi rimane invariata

Caledor commented:
si quella ok ma io parlavo di portare gli altri "bluetooth" minuscoli sparsi per il repository a "Bluetooth"

ale8530 commented:
Lo vediamo dopo, per questo dobbiamo fare piccole PR, troppe stringhe sono state fatte in questa PR e va fatto una verifica

Caledor committed:
Fix PR

"Temperatura e saturazione colori" è diventato "Temperatura colore e
saturazione" per coerenza con il cambiamento già discusso

Caledor commented:
c'è tutto. l'unico cambiamento extra te l'ho motivato nella descrizione del commit e l'ho propagato a tutti gli altri settings.

ale8530 commented:
Stoppare le modifiche fino al merge :)
